build_tools/cmake/iree_benchmark_suite.cmake

```diff
@@ -38,6 +38,8 @@
 #   DRIVER: The runtime driver.
 #   RUNTIME_FLAGS: A list of command-line options and their values to pass
 #       to the IREE runtime during benchmark exectuion.
+#   COLLECT_COMPILATION_STATISTICS: Indidate whether to collect compilation
+#       statistics.
```

Comment:
Can you elaborate on what collection actually does? (writes a JSON file to disk when the target is built? Updates a database entry when a binary is ran?)